hs_core/hydroshare/utils.py

```diff
@@ -322,6 +322,26 @@ def get_resource_file_extension(res_file):
     return fl_ext
 
 
+def get_resource_file_name_and_extension(res_file):
```

Comment:
@pkdash there were some occurrences of "get_resource_file_extension" in netcdf and timeseries. I replaced them with "get_resource_file_name_and_extension(res_file)[1]"